packages/color-area/test/color-area.test.ts

```diff
@@ -0,0 +1,272 @@
+/*
+Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import { fixture, elementUpdated, expect, html } from '@open-wc/testing';
+import {
+    shiftEvent,
+    arrowUpEvent,
+    arrowDownEvent,
+    arrowLeftEvent,
+    arrowRightEvent,
+    arrowUpKeyupEvent,
+    arrowDownKeyupEvent,
+    arrowLeftKeyupEvent,
+    arrowRightKeyupEvent,
+    shiftKeyupEvent,
+} from '../../../test/testing-helpers.js';
+
+import '../sp-color-area.js';
+import { ColorArea } from '..';
+
+describe('ColorArea', () => {
+    it('loads default color-area accessibly', async () => {
+        const el = await fixture<ColorArea>(
+            html`
+                <sp-color-area></sp-color-area>
+            `
+        );
+
+        await elementUpdated(el);
+
+        await expect(el).to.be.accessible();
+    });
+    it('accepts "color" values as hsl', async () => {
```

Comment:
We aren't using shared processing for the color value, so it would be good if we could include this level of input output testing on all three: https://github.com/adobe/spectrum-web-components/blob/halsema/colour-feature/packages/color-slider/test/color-slider.test.ts#L406 or review whether an abstraction theres is possible.